src/shared/model/ClientState.ts

```diff
@@ -41,6 +41,8 @@ export interface PageData {
   previousPage?: ConsentPath;
   // reset password token specific
   timeUntilTokenExpiry?: number;
+  firstName?: string;
+  secondName?: string;
```

Comment:
maybe a comment here explaining this is for jobs specifically e.g. 
```suggestion
  // jobs specific flow
  firstName?: string;
  secondName?: string;
```